src/worker/ingestion/team-manager.ts

```diff
@@ -146,7 +146,12 @@ export class TeamManager {
             const distinctIds: { distinct_id: string }[] = organizationMembers.rows
             for (const { distinct_id } of distinctIds) {
                 posthog.identify(distinct_id)
-                posthog.capture('first team event ingested', { team: team.uuid })
+                posthog.capture('first team event ingested', {
+                    team: team.uuid,
+                    sdk: properties.$lib,
+                    realm: properties.realm,
```

Comment:
Do events actually have a `realm` property? I haven't seen it

Reply:
I ran the plugin server locally and was seeing realm show up on props, so I believe so – at least for some events

Reply:
<img width="978" alt="Screen Shot 2021-04-15 at 8 20 00 AM" src="https://user-images.githubusercontent.com/5965891/114894327-65616f80-9dc3-11eb-8af6-1ca0118e6fb4.png">